lib/Doctrine/ORM/QueryBuilder.php

```diff
@@ -490,7 +494,7 @@ public function getAllAliases()
     }
 
     /**
-     * Gets the root entities of the query. This is the entity aliases involved
+     * Gets the root entities of the query. This is the entity classes involved
```

Comment:
:thinking: maybe this was referring to the deprecated but still existing class alias feature? https://github.com/doctrine/orm/issues/8818

I don't know if those are resolved at that point.